install_config/master_node_configuration.adoc

```diff
@@ -1422,6 +1410,34 @@ start on a machine that does not have docker started.
 
 |===
 
+[[node-config-volume-config]]
+=== Volume Configuration
+
+You can use the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/storage_administration_guide/ch-xfs[XFS quota subsystem] 
+to limit the size of `emptyDir` volumes  and volumes based on an `emptyDir` volume, such as secrets and configmaps, on each node.
+
+Enable a quota on local storage use per unique FSGroup ID usin g the *_/var/lib/origin/openshift.local.volumes/volume-config.yaml_* file.
+
+----
+apiVersion: kubelet.config.openshift.io/v1
+kind: VolumeConfig
+localQuota: <1>
+  perFSGroupInGiB: 1 <2>
+---
+
+<1> Contains options for controlling local volume quota on the node.
+<2> Set this value to a resource quantity representing the desired quota per 
```

Comment:
Due to some complications in the code, this value is now numeric, not a resource quantity.  It is the quota in GBs.

Reply:
@sjenning Is this setting GB (gigabyte) or GiB (gibibyte) as the parameter is `perFSGroupInGiB.